tracer/src/Datadog.Trace/Agent/AgentWriter.cs

```diff
@@ -135,6 +140,48 @@ public void WriteTrace(ArraySegment<Span> trace)
             }
         }
 
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        private void LogSpanDebugInformation(TraceContext traceContext, ArraySegment<Span> trace)
+        {
+            foreach (var span in trace)
+            {
+                Log.Debug(
```

Comment:
I don't think we need to include `SamplingPriority` here for each span if it's already included in `LogRootSpanDebugInformation()`, since it's a single value for the entire trace.